hal/src/main/java/edu/wpi/first/hal/HALUtil.java

```diff
@@ -76,14 +54,6 @@ public final class HALUtil extends JNIWrapper {
    */
   public static native String getSerialNumber();
 
-  /**
-   * Returns the comments from the roboRIO web interface.
-   *
-   * @return The comments string.
-   * @see "HAL_GetComments"
-   */
-  public static native String getComments();
```

Comment:
I think the plan is to actually keep something like this API. It was useful, and can be added to the web UI.